### ISSUE TITLE
getRealPosition Issues with infiniteAdapter

### ISSUE THREAD

**Issue by divysoni3** [yarolegovich/DiscreteScrollView#125]
While getting real position from selected position it gives wrong value for second position. And it happens while scrolling from right to left it works fine with opposite scrolling.

@yarolegovich I have produced the problem in your sample app, go to your layout `item_shop_card `set width height of cardView 100dp and margin 0dp.

```
<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="100dp"
    android:layout_height="100dp"
    android:layout_margin="0dp"
    android:layout_gravity="center">

    <ImageView
        android:id="@+id/image"
        android:scaleType="centerCrop"
        android:layout_width="match_parent"
        android:layout_height="match_parent" />

</android.support.v7.widget.CardView>
```
Now you can see the same problem.
Please give the solution for this.

**Comment by yarolegovich** [comment]
Clone please [this branch](https://github.com/yarolegovich/DiscreteScrollView/compare/125-infinite-adapter-issue) and check whether the issue is resolved.

**Comment by divysoni3** [comment]
Yes it solved my problem, Thanks a lot @yarolegovich  👍

**Comment by yarolegovich** [comment]
The fix is in `1.4.9`.
